catalog/to-consul/resource_test.go

```diff
@@ -994,9 +727,8 @@ func TestServiceResource_nodePort_singleEndpoint(t *testing.T) {
 	require.Equal("k8s-sync", actual[0].Node)
 }
 
-// Test that a NodePort created earlier works (doesn't require an Endpoints
-// update event).
-func TestServiceResource_nodePortInitial(t *testing.T) {
```

Comment:
Can you tell me which test this was duplicating so it's easier for me to review

Reply:
Yep, good point! `TestServiceResource_nodePortInitial` is a duplicate of `TestServiceResource_nodePort`